.travis.yml

```diff
@@ -1,25 +1,6 @@
 import:
-  - source: Alfresco/alfresco-build-tools:.travis.maven_config.yml@v1.1.2
-  - source: Alfresco/alfresco-build-tools:.travis.java_config.yml@v1.1.2
-  - source: Alfresco/alfresco-build-tools:.travis.common.yml@v1.1.2
-  - source: Alfresco/alfresco-build-tools:.travis.pre-commit.yml@v1.1.2
+  - source: Alfresco/alfresco-build-tools:.travis.java.yml@v1.1.5
   - source: Alfresco/alfresco-process-scripts:slack_apa_channel_notification.yml@master
-jobs:
```

Comment:
Perfect. Since maven itself is generating the code as part of the build, we don't need to override those stages any more

Reply:
Exactly, that's the idea